SparkPost/matchbox
# UX-607 Setup typescript in matchbox-icons

### What Changed

- Sets up typescript in matchbox-icons
- Changes output directory to `./dist`
- Sets up root-level jest for typescript

### How To Test or Verify

- Visit matchbox-icons and verify `npm run build` works
- Run root-level tests `npm run test:unit`

### PR Checklist
